uberfire-security/uberfire-security-management/uberfire-security-management-backend/src/main/java/org/uberfire/ext/security/management/BackendUserSystemManager.java

```diff
@@ -92,6 +91,8 @@ public void initialize() throws Exception {
                     isUserManagerActive = true;
                 } catch (ClassCastException e) {
                     // Manager is not contextual.
```

Comment:
It's not related to the PR, but swallow this exception make me nervous... wdyt to add a log here?